src/sections/maibornwolff.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ojemba</title>
+</head>
+<body>
+    <section class="pb-40">
```

Comment:
- [ ] kindly consider giving this section the same oblique borders on top and bottom to make your design an exact duplicate of the given one in Figma 
hint 😀  you can download the background from figma and directly use it to minimize work